### PR TITLE
hard-coded '100' caused garbled map when MAX_MOBS < 100; this is not …

### PR DESCRIPTION
**Pull request by efexgee**
bug fix

### CODE REVIEW

File: game.c [hunk 1]
```diff
@@ -108,13 +108,7 @@ void setup_level() {
         mobs[i].stacks = false;
     }
 
-    player = &mobs[MAX_MOBS-1];
```
Comment (alec-deason) [inline game.c:111]: How is the player mob being defined? Doesn't this straight up break the game?

File: game.c [hunk 2]
```diff
@@ -108,13 +108,7 @@ void setup_level() {
...
-    player->x = player->y = 1;
-    player->behavior = KeyboardInput;
-    player->display = ICON_HUMAN;
-    player->active = true;
-
-    for (int i=0; i < 100; i++) {
+    for (int i=0; i < MAX_MOBS; i++) {
```
Comment (alec-deason) [inline game.c:111]: I guess the actual mob initialization will normally happen during level creation so this loop will disappear soon anyway.

**Review by alec-deason — CHANGES_REQUESTED** [review]

File: game.c (hunk 1 above)
Comment (efexgee) [reply game.c:111]: If this was a question for me, then this is because I screwed up my repo. I didn't write that line.

**Comment by efexgee** [comment]
Closing this because it was on master and my repo got all sideways.

**Pull request closed without merging**